test/ethyl/lint_test.exs

```diff
@@ -0,0 +1,79 @@
+defmodule Ethyl.LintTest do
+  use ExUnit.Case, async: true
+
+  defmacrop assert_expanded(lhs, do: rhs) do
+    quote do
+      lhs =
+        unquote(lhs)
+        |> Ethyl.Lint.expand_aliases()
+        |> strip_meta()
+
+      rhs = strip_meta(unquote(Macro.escape(rhs)))
+
+      assert lhs == rhs
+    end
+  end
+
+  describe "expand_aliases/1 correctly expands" do
+    test "given a simple fixture using alias/1" do
+      fixture =
+        quote do
+          alias File.Stream
+          Stream.foo()
+        end
+
+      assert_expanded fixture do
+        :ok
+        File.Stream.foo()
+      end
+    end
+
+    test "given a simple fixture using alias/2" do
+      fixture =
+        quote do
+          alias Application, as: App
+          App.foo()
+        end
+
+      assert_expanded fixture do
+        :ok
+        Application.foo()
+      end
+    end
+
+    test "given a fixture that aliases multiple times" do
+      fixture =
+        quote do
+          alias Foo, as: F
+          alias F, as: B
+          B.foo()
+        end
+
+      assert_expanded fixture do
+        :ok
+        :ok
```

Comment:
as you can see in lines 19 & 24 in the lib code, we replace the `alias` ast with `:ok` which is just a value, so we need to add some `:ok`s to our expected expanded code here so that the ASTs match